install/postgres.sh

```diff
@@ -16,6 +16,10 @@ if [ $OS_CODENAME = "jessie" ]
 then
     APT_REPO="apt-archive.postgresql.org"
 fi
+if [ $OS_CODENAME = "stretch" ]
```

Comment:
Why not use an `OR` with the test above ?

Reply:
not so used to bash scripts and need to have a quick solution

Reply:
```if [ $OS_CODENAME = "jessie" ] || [ $OS_CODENAME = "stretch" ]```

Reply:
done in last commit